### DISCUSSION TITLE
Outline/Add Border to Screen Capture

### DISCUSSION THREAD

**Discussion by shockogit** [flameshot-org/flameshot#3752] [Ideas]
It would be great if the tool could add a border to screenshot images like ShareX does.

**Comment by steveklett** [comment]
Snag-It also does this and it's very useful

**Comment by mmahmoudian** [comment]
This is duplicate of #690. For X11 linux you can checkout the proposed solution, but in general this type of post-screenshot modification is left to be done by plugins. You can checkout [the Plugin label](https://github.com/flameshot-org/flameshot/issues?q=label%3APlugin+) for features that are lined  up for plugins. Please consider giving thumbs-up to the features you like.
